src/runtime/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kata-containers/kata-containers/src/runtime
 
-go 1.21
+go 1.22.11
```

Comment:
cc @stevenhorsman who is working on a toolchain bump in a separate PR. It looks `1.22.11` may not be the real requirement here (but just Go version `1.22.0`)?

Reply:
Yeah - I think  the minimum go versions are safe to use the `.0` minor release version as there shouldn't be any API incompatibility in minor versions. I'm not an expert here though!

Reply:
Thanks. I fix to `1.22`. With `go vendor`, `go mod tidy` and `make`, only the `go.mod` and `go.sum` have been changed. I am not an expert of golang. Does this make sense?